mods/cnc/weapons/smallcaliber.yaml

```diff
@@ -114,6 +114,17 @@ MachineGun:
 	Warhead@2Eff: CreateEffect
 		Explosions: piffs
 
+MachineGunH:
```

Comment:
Optional: You can even reduce the amount of lines by using the inheritance feature.  
```yaml
MachineGunH:
  Inherits: MachineGun
  Warhead@1Dam: SpreadDamage
    Versus:
      Light: 55
```